src/consts/config.ts

```diff
@@ -15,7 +15,6 @@ interface Config {
   withdrawalWhitelist: string; // comma-separated list of CAIP2 chain IDs to which transfers are supported
   transferBlacklist: string; // comma-separated list of routes between which transfers are disabled. Expects Caip2Id-Caip2Id (e.g. ethereum:1-sealevel:1399811149)
   enableExplorerLink: boolean; // Include a link to the hyperlane explorer in the transfer modal
-  enableAutoTokenSelection: boolean; // Automatically select the first valid for the token field
```

Comment:
This is no longer needed, the token select field is now smart enough to auto-select when there's 1, otherwise enable the list modal